app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt

```diff
@@ -113,17 +116,19 @@ class ImportSpotifyPlaylistsActivity : AppCompatActivity() {
             override fun onResponse(call: Call, response: Response) {
                 val parsedResponseString = response.body()?.string()
                 val parsedResponse = parser.parse(parsedResponseString)
-                var playlistInfo = mutableSetOf<Song>()
-                println(parsedResponse.toString())
-                if (parsedResponse == null || parsedResponse.has("error")){
-
-                }
-                else {
+                val playlistInfo = mutableSetOf<Song>()
+                if (!(parsedResponse == null || parsedResponse.has("error"))) {
                     val songs = parsedResponse.getJSONArray("items")
                     for (i in 0 until songs.length()){
-                        var songObject = songs.getJSONObject(i).getJSONObject("track")
+                        val songObject =
+                        try {
+                            songs.getJSONObject(i).getJSONObject("track")
+                        } catch (e : JSONException){
+                            // TODO: handle error
+                            continue
+                        }
```

Comment:
Do you plan on doing this in a later sprint or did you forget about it?

Reply:
No this was not handled at all previously, that is the purpose of this PR, to do a provisonal solution. Nathan may be responsible for it in another sprint.